.github/workflows/ci.yml

```diff
@@ -138,3 +138,32 @@ jobs:
           make test
           make
         working-directory: src/github.com/containerd/imgcrypt
+
+  windows-mingw:
+    name: Windows Tests
```

Comment:
nit... I'd call it Windows mingw32 Testing

will need Kevin to chime in on what if any windows tooling also needs to be obfuscated while go build is running to setup a reproduction to typical windows environments outside this runner

Reply:
Sure, I can rename this. For now we need to figure out what actually the culprit is that breaks the containerd build. I am not actually sure whether we gain any insights with this additional build...